docs/script to save certificates.md

```diff
@@ -0,0 +1,42 @@
+# Scripts to save certificates
+
+## While dismantling our environments sometimes we need to save the Letsencrypt certificate issued for the domain name.
+
+1. Script such that it can be run from a client machine that will zip the certs on host and download them to the client  machine .
+2. Reverse script to upload the certificates at the right place before starting to install.
+
+
+**1. Script such that it can be run from a client machine that will zip the certs on host and download them to the client  machine .**
+
+* Script to download certificates from host machine to client machine
```

Comment:
please remove image from here and add only the required procedure to run the script